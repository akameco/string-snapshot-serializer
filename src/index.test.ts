import { getSnapshotSerializer } from '.'

function a(input: string): string {
  return `test "${input}"`
}

test('snapshot default', () => {
  expect(a('unicorn')).toMatchSnapshot()
})

test('snapshot with serializer', () => {
  expect.addSnapshotSerializer(getSnapshotSerializer())
  expect(a('unicorn')).toMatchSnapshot()
})
