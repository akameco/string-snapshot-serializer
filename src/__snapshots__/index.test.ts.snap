// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`snapshot default 1`] = `"test \\"unicorn\\""`;

exports[`snapshot with serializer 1`] = `test "unicorn"`;
